pkg/operator/ceph/object/spec.go

```diff
@@ -232,21 +236,21 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) v1.Container {
 		mount := v1.VolumeMount{Name: certVolumeName, MountPath: certDir, ReadOnly: true}
 		container.VolumeMounts = append(container.VolumeMounts, mount)
 	}
-	if c.clusterSpec.Security.KeyManagementService.IsEnabled() {
+	if kmsEnabled, _ := c.CheckRGWKMS(); kmsEnabled {
```

Comment:
Why is the error ignored?

Reply:
@leseb : currently `makeDaemonContainer()` does not return any error, hence I ignore this error. The parent function `makeRGWPodSpec()` calls `checkKMS` before volume mounts and handles the same error. If need I can put a log message for the error

Reply:
The result of `c.CheckRGWKMS()` can change between the parent and the current caller since we are calling the API in `ValidateConnectionDetails`. So it's better to evaluate the error and return if `!= nil`.